APCAppCore/APCAppCore/Library/Categories/ORKOrderedTask+APCHelper.m

```diff
@@ -105,7 +105,7 @@ + (ORKOrderedTask *)twoFingerTappingIntervalTaskWithIdentifier:(NSString *)ident
         case APCTapHandOptionRight:
             rightHand = YES; break;
         default:
-            rightHand = (((NSUInteger)[NSDate timeIntervalSinceReferenceDate])%2 == 0); break;
+            rightHand = (arc4random()%2 == 0); break;
```

Comment:
use arc4random_uniform(2) to avoid "modulo bias" (https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man3/arc4random_uniform.3.html)